src/plugins/action_generate_review_assignment_records/src/generateReviewAssignments.ts

```diff
@@ -116,6 +109,111 @@ module.exports['generateReviewAssignments'] = async function (input: any, DBConn
   }
 }
 
+interface GenerateNextReviewAssignmentsProps {
+  db: any
+  applicationId: number
+  templateId: number
+  nextReviewLevel: number
+  nextStageNumber: number
+  nextStageId: number
+  isLastLevel: boolean
+}
+
+const generateNextReviewAssignments = async ({
+  db,
+  applicationId,
+  templateId,
+  nextReviewLevel,
+  nextStageNumber,
+  nextStageId,
+  isLastLevel,
+}: GenerateNextReviewAssignmentsProps) => {
+  // TODO: Needs to check if there are already reviews created - if partial review
+  // for level - 1 are submitted separetedily will create 2 reviewAssignments for level 1++
```

Comment:
I'm not sure I follow what you're saying here. It won't create additional reviewAssignments, it'll just update existing ones.

Reply:
Cool. I'll remove this comment